lib/utils/analytics/index.js

```diff
@@ -180,8 +174,15 @@ async function sendPending(options = {}) {
         return null;
       })
     )
+  ).filter(Boolean);
+
+  return request(
```

Comment:
In previous version we also had `if (!analyticsUrl) return null` prior calling `request`, I think it's safer to keep it

Reply:
good catch - in this case I think we should do it at the beginning of processing in `sendPending`